components/user_settings/advanced/user_settings_advanced.jsx

```diff
@@ -85,6 +86,11 @@ export default class AdvancedSettingsDisplay extends React.PureComponent {
         };
     }
 
+    componentDidMount() {
+        const prev = this.isDeactivateEnabled() ? 'deactivateAccount' : 'advancedPreviewFeatures';
```

Comment:
Worth throwing `this.isDeactivateEnabled() ? 'deactivateAccount' : 'advancedPreviewFeatures'` in its own method? I see it being used in one other place

Reply:
just trying to make explicit the intent, I can replace with its contents if you prefer that way.

Reply:
0/5, I'll leave it up to you